tracing-core/src/test/groovy/io/micronaut/tracing/instrument/util/MDCReactorSpec.groovy

```diff
@@ -167,7 +167,7 @@ class MDCReactorSpec extends Specification {
     }
 
     @Filter("/mdc/test**")
-    @Requires(property = 'mdc.reactortest.enabled')
+    @Requires(property = 'mdc.reactortest.enabled', value = "true")
```

Comment:
replace `value = "true"` with `value = StringUtils.TRUE`. Why do we need to set it in MN 4?